custom_components/kia_uvo/Vehicle.py

```diff
@@ -292,8 +300,10 @@ def set_engine_type(self):
                     self.engine_type = VEHICLE_ENGINE_TYPE.IC
         _LOGGER.debug(f"{DOMAIN} - Engine type set {self.engine_type}")
 
-    def get_child_value(self, key):
-        value = self.vehicle_data
+    def get_child_value(self, key, source=None, default=None):
```

Comment:
can we have default valie as `self.vehicle_data`? I do not know if this can be done.

Reply:
not sure what you mean here.